modules/local/collectgenomes.nf

```diff
@@ -0,0 +1,58 @@
+process COLLECTGENOMES {
+    tag "$accno"
+    label 'process_single'
+
+    conda "conda-forge::wget=1.18"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gnu-wget:1.18--h36e9172_9':
+        'biocontainers/gnu-wget:1.18--h36e9172_9' }"
```

Comment:
Why not use the ubuntu20,04 container as used above to reduce container HDD usage